docs/src/pages/demos/progress/CircularIntegration.tsx

```diff
@@ -60,7 +60,7 @@ function CircularIntegration() {
     if (!loading) {
       setSuccess(false);
       setLoading(true);
-      timer.current = setTimeout(() => {
```

Comment:
Why is this change needed?

Reply:
The typescript compiler claims it's from Node, which returns a different type.
VSCode, depending on the position of the stars, claims either Node or window